Sources/Core/Microsoft.StreamProcessing/Operators/Afa/AfaInterfaces.cs

```diff
@@ -143,62 +159,75 @@ public interface IAbstractPatternRoot<TKey, TPayload, TRegister, TAccumulator>
         /// </summary>
         /// <param name="condition"></param>
         /// <param name="aggregator"></param>
         /// <returns></returns>
         IPattern<TKey, TPayload, TRegister, TAccumulator> ListElement(Expression<Func<long, List<TPayload>, TRegister, bool>> condition = null, Expression<Func<long, List<TPayload>, TRegister, TRegister>> aggregator = null);
+
+
         /// <summary>
         /// Interface is only public to serve CLR strong typing - instances should only be created internally. Do not use directly.
         /// </summary>
         /// <param name="condition"></param>
         /// <param name="aggregator"></param>
         /// <returns></returns>
         IPattern<TKey, TPayload, TRegister, TAccumulator> ListElement(Expression<Func<long, List<TPayload>, TRegister, bool>> condition, Expression<Func<List<TPayload>, TRegister, TRegister>> aggregator);
+
         /// <summary>
         /// Interface is only public to serve CLR strong typing - instances should only be created internally. Do not use directly.
         /// </summary>
         /// <param name="condition"></param>
         /// <param name="aggregator"></param>
         /// <returns></returns>
         IPattern<TKey, TPayload, TRegister, TAccumulator> ListElement(Expression<Func<long, List<TPayload>, TRegister, bool>> condition, Expression<Func<List<TPayload>, TRegister>> aggregator);
+
         /// <summary>
         /// Interface is only public to serve CLR strong typing - instances should only be created internally. Do not use directly.
         /// </summary>
         /// <param name="condition"></param>
         /// <param name="aggregator"></param>
         /// <returns></returns>
         IPattern<TKey, TPayload, TRegister, TAccumulator> ListElement(Expression<Func<List<TPayload>, bool>> condition, Expression<Func<long, List<TPayload>, TRegister, TRegister>> aggregator = null);
+
         /// <summary>
         /// Interface is only public to serve CLR strong typing - instances should only be created internally. Do not use directly.
         /// </summary>
         /// <param name="condition"></param>
         /// <param name="aggregator"></param>
         /// <returns></returns>
         IPattern<TKey, TPayload, TRegister, TAccumulator> ListElement(Expression<Func<List<TPayload>, bool>> condition, Expression<Func<List<TPayload>, TRegister, TRegister>> aggregator);
+
+
         /// <summary>
         /// Interface is only public to serve CLR strong typing - instances should only be created internally. Do not use directly.
         /// </summary>
         /// <param name="condition"></param>
         /// <param name="aggregator"></param>
         /// <returns></returns>
         IPattern<TKey, TPayload, TRegister, TAccumulator> ListElement(Expression<Func<List<TPayload>, bool>> condition, Expression<Func<List<TPayload>, TRegister>> aggregator);
+
+
```

Comment:
Why the double spaces? Let's standardize on only single spaces if possible.

Reply:
Accidental - Find&Replace mistake

---
In reply to: [286226139](https://github.com/microsoft/Trill/pull/94#discussion_r286226139) [](ancestors = 286226139)